plugins/ldap/pkg/app/fetch.go

```diff
@@ -16,7 +16,7 @@ type FetchCmd struct {
 	UserFilter  string `short:"f" help:"LDAP user filter" env:"LDAP_USER_FILTER" default:"(&(objectClass=organizationalPerson))"`
 	GroupFilter string `short:"g" help:"LDAP group filter" env:"LDAP_GROUP_FILTER" default:"(&(objectClass=groupOfNames))"`
 	Insecure    bool   `short:"i" help:"Allow insecure LDAP connection" env:"LDAP_INSECURE" default:"false"`
-	UUIDField   string `short:"U" help:"LDAP field to use as UUID" env:"LDAP_UUID_FIELD" default:"objectGUID"`
+	IDField     string `short:"U" help:"LDAP field to use as UUID" env:"LDAP_UUID_FIELD" default:"objectGUID"`
```

Comment:
The description should be updated too. "LDAP field to use as the ID for directory objects".

Reply:
And the env var name should be updated too